### DISCUSSION TITLE
Love the solution you provided, have you ever tried converting PPTX slides?

### DISCUSSION THREAD

**Discussion by alexanderames** [slidevjs/slidev#70] [Q&A]
Really love the work here. The problem I need to solve is parsing PPT/PPTX slides and presenting them, while maintaining their animations. Right now I convert from PowerPoint into a PDF, but none of the animations transfer over.

**Comment by antfu** [comment]
Probably not on our roadmap, as this could take a lot of efforts to generate the correct format for PPT to consume. And even if we somehow managed to do it, you are still loosing a lot of features like components, interactivity, inline editor and some other cool stuff that are only available with web technology. I kinda think jump out of the "compatible file formats" allowing us to explore much more creativity and expressiveness on what the presentations could apply with. Thanks for bringing this up
